index.js

```diff
@@ -100,7 +102,7 @@ function createDat (dirOrStorage, opts, cb) {
       archive.on('error', cb)
       archive.ready(function () {
         debug('archive ready. version:', archive.version)
-        if (archive.metadata.has(0) && archive.version) archive.resumed = true
```

Comment:
I wonder if we need to keep this around for archives that use other storage (not in `.dat` folder)?

Reply:
I'm not able to test accessing the same storage using ```temp``` or ```ram``` because by definition they will no longer be accessible after closing, right? So I'll keep the same check for metadata and version alongside the ```.dat``` file for now.